crypto/rsa/rsa_pmeth.c

```diff
@@ -340,10 +342,9 @@ static int pkey_rsa_decrypt(EVP_PKEY_CTX *ctx,
         ret = RSA_private_decrypt(inlen, in, out, ctx->pkey->pkey.rsa,
                                   rctx->pad_mode);
     }
-    if (ret < 0)
-        return ret;
-    *outlen = ret;
-    return 1;
+    *outlen = constant_time_select_s(constant_time_msb_s(ret), *outlen, ret);
```

Comment:
There is no constant_time_select_s and constant_time_msb_s in 1.1.0 and 1.0.2 so that will only
cherry-pick to 1.1.1

Reply:
I guess this can result in undefined behaviour, *outlen might not be initialized ...